charters/wot-wg-charter-draft-2019.html

```diff
@@ -70,17 +69,23 @@
     <main>
       <h1 id="title">Web of Things Working Group Charter</h1>
 
-      <p class="mission">The <strong>mission</strong> of the <a href="">
-	      Web of Things Working Group</a> is to 
-	      counter the fragmentation of the IoT through standard complementing building blocks 
-	      (e.g., metadata and APIs) that enable easy integration across IoT platforms and application domains.
-	      This Working Group Charter covers those aspects that the 
-	      <a href="http://w3c.github.io/wot/charters/wot-ig-2019.html">Web of Things Interest Group</a> 
-	      believes are mature enough to progress to W3C Recommendations. 
-	      <!--
-	      Further background is available in the accompanying 
-	      <a href="http://w3c.github.io/wot/charters/wot-white-paper-2016.html">white paper</a>.
-              -->
+      <p class="mission">
+        The <strong>mission</strong> of the 
+        <a href="https://www.w3.org/WoT/WG/">Web of Things Working Group</a> 
+        is to counter the fragmentation of the IoT through 
+        the specification of
+        standard complementing building blocks (e.g., metadata and APIs) 
```

Comment:
I would leave out "standard complementing" ore reword this line to
"building blocks that complement existing standards"

Reply:
Yeah, I never liked that wording, it was borrowed from the last charter and is confusing.   I'll reword as you suggest.

Reply:
By the way I was really thinking that we should add a paragraph or two to the introduction giving some example use cases where the WoT work is useful for particular sets of users, similar to what we discussed with the TAG.

Reply:
Reworded.